sonar-java-plugin/src/test/java/org/sonar/plugins/java/JavaSquidSensorTest.java

```diff
@@ -103,7 +108,7 @@ private void testIssueCreation(InputFile.Type onType, int expectedIssues) throws
     JavaSquidSensor jss = new JavaSquidSensor(sonarComponents, fs, javaResourceLocator, settings.asConfig(), noSonarFilter, postAnalysisIssueFilter);
 
     jss.execute(context);
-    verify(noSonarFilter, times(1)).noSonarInFile(fs.inputFiles().iterator().next(), Sets.newHashSet(91));
+    verify(noSonarFilter, times(1)).noSonarInFile(fs.inputFiles().iterator().next(), Sets.newHashSet(96));
```

Comment:
@benzonico I fail to understand this change

Reply:
This test relies on its own source. And so the line number as been change because of some imports .